lib/kubeclient/common.rb

```diff
@@ -254,6 +241,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace = nil, container: nil, previous: false)
+      params = { params: {} }
+      params[:params][:previous] = true if previous
+      params[:params][:container] = container if container
+
+      ns = build_namespace_prefix(namespace || 'default')
```

Comment:
Actually, given how the logs endpoint is designed, it's impossible for it to succeed if you don't specify a namespace.  The URL for the endpoint is `/api/v1/namespaces/{namespace}/pods/{name}/log`.

So what do you think of me changing the method signature to this?

```ruby
def get_pod_log(pod_name, namespace:, container: nil, previous: false)
  ...
end
```

That acts as documentation that it's required, and if you don't specify it, you'll immediate get:

`ArgumentError: missing keyword: namespace`